platform/ui/src/components/ViewportActionArrows/ViewportActionArrows.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Icon from '../Icon';
+
+const arrowClasses =
+  'cursor-pointer flex items-center justify-center shrink-0 text-primary-light active:text-white hover:bg-secondary-light/60';
+
+type ViewportActionArrowsProps = {
+  onArrowsClick: (arrow: string) => void;
+};
+
+function ViewportActionArrows({ onArrowsClick }: ViewportActionArrowsProps) {
+  return (
+    <div className="invisible flex group-hover:visible">
```

Comment:
I'm not sure why this is invisible

Reply:
I will add a comment, but according to the new design discussed, the action arrows only appear when hovering over the viewport.

Reply:
Let's review this design whenever I add the window level menu as part of the work for #3743. Sound good?